### ISSUE TITLE
Any samples from current training ?

### ISSUE THREAD

**Issue by rishikksh20** [scutcsq/Neural-Transducers-for-Two-Stage-Text-to-Speech-via-Semantic-Token-Prediction#1]
Hi @scutcsq , saw your repo and tracking your training, have you able to generate some good quality speech ? 
Please share some samples or pretrained model if possible. 
Thanks for code.

**Comment by scutcsq** [comment]
Hi @rishikksh20 , I still face the problem and I am still working on to fix it. When I fix this problem, I will upload the samples.

**Comment by rishikksh20** [comment]
Hi What kind pf problem you are facing I might be help you ?

**Comment by scutcsq** [comment]
Thank you very much! I found it difficult to generate the comprehensible speech from the transducer model. I have made sure the T2S model works normally. I speculate that the transducer model didn't converge well. No matter if I use k2.rnnt_loss_simple or k2.rnnt_loss_smoothed, the model is always slow to converge, and even after 3 or 4 days of training, the loss function per sample is still high on average (100 or so), I'm not quite sure if this phenomenon is normal. I haven't trained a transducer before, It would be great if you could provide some advice on training transducer models!

**Comment by mondorysix** [comment]
> Thank you very much! I found it difficult to generate the comprehensible speech from the transducer model. I have made sure the T2S model works normally. I speculate that the transducer model didn't converge well. No matter if I use k2.rnnt_loss_simple or k2.rnnt_loss_smoothed, the model is always slow to converge, and even after 3 or 4 days of training, the loss function per sample is still high on average (100 or so), I'm not quite sure if this phenomenon is normal. I haven't trained a transducer before, It would be great if you could provide some advice on training transducer models!

May I ask if the issue you mentioned has been resolved now? Could it be related to the size of the dataset? I also encountered difficulties fitting the model during stage 1 of training GPTsoVits, which has troubled me for a long time.

**Comment by scutcsq** [comment]
> > Thank you very much! I found it difficult to generate the comprehensible speech from the transducer model. I have made sure the T2S model works normally. I speculate that the transducer model didn't converge well. No matter if I use k2.rnnt_loss_simple or k2.rnnt_loss_smoothed, the model is always slow to converge, and even after 3 or 4 days of training, the loss function per sample is still high on average (100 or so), I'm not quite sure if this phenomenon is normal. I haven't trained a transducer before, It would be great if you could provide some advice on training transducer models!
> 
> May I ask if the issue you mentioned has been resolved now? Could it be related to the size of the dataset? I also encountered difficulties fitting the model during stage 1 of training GPTsoVits, which has troubled me for a long time.
Sorry, I am busy before and have no time to solve this problem. These days I will retrain the model and try to solve the problem. I use the new training method from vall-e and you can see the dynamic version. The loss is small, but when inferencing, the model has some problem.

**Comment by mondorysix** [comment]
> > > Thank you very much! I found it difficult to generate the comprehensible speech from the transducer model. I have made sure the T2S model works normally. I speculate that the transducer model didn't converge well. No matter if I use k2.rnnt_loss_simple or k2.rnnt_loss_smoothed, the model is always slow to converge, and even after 3 or 4 days of training, the loss function per sample is still high on average (100 or so), I'm not quite sure if this phenomenon is normal. I haven't trained a transducer before, It would be great if you could provide some advice on training transducer models!
> > 
> > 
> > May I ask if the issue you mentioned has been resolved now? Could it be related to the size of the dataset? I also encountered difficulties fitting the model during stage 1 of training GPTsoVits, which has troubled me for a long time.
> > Sorry, I am busy before and have no time to solve this problem. These days I will retrain the model and try to solve the problem. I use the new training method from vall-e and you can see the dynamic version. The loss is small, but when inferencing, the model has some problem.

Thank you for your response and I hope it works. By the way I feel that the dataset might not be large enough, as I used 85 hours of data and still failed. I will continue to try, and I will let you know if I make any new discoveries.
